Resources/Prototypes/Reagents/Consumable/Food/food.yml

```diff
@@ -15,23 +15,6 @@
   - !type:PlantAdjustHealth
     amount: 0.5
 
-- type: reagent
-  id: Glucose
-  name: glucose
-  desc: A simple sugar found in many foods.
-  physicalDesc: syrupy
-  color: "#ffffff"
-  boilingPoint: 340282300000000000000000000000000000000 #Fun fact: Glucose can't boil. So let's just set it to the maximum float value.
-  meltingPoint: 146.0
-  plantMetabolism:
-  - !type:PlantAdjustNutrition
-    amount: 0.1
-  - !type:PlantAdjustWeeds
-    amount: 2
-  - !type:PlantAdjustPests
-    amount: 2
```

Comment:
Shouldn't this be added to sugar if you remove Glucose? Is there a specific reason sugar shouldn't have those?

Reply:
I added it back